drivers/filters/mountmgr/mountmgr.c

```diff
@@ -89,28 +89,19 @@ HasDriveLetter(IN PDEVICE_INFORMATION DeviceInformation)
     PLIST_ENTRY NextEntry;
     PSYMLINK_INFORMATION SymlinkInfo;
 
-    /* To have a drive letter, a device must have symbolic links */
-    if (IsListEmpty(&(DeviceInformation->SymbolicLinksListHead)))
-    {
-        return FALSE;
-    }
-
-    /* Browse all the links untill a drive letter is found */
-    NextEntry = &(DeviceInformation->SymbolicLinksListHead);
-    do
+    /* Browse all symlinks to check if there is at least a drive letter */
```

Comment:
"the" is legit here too

Reply:
Re-added. (Suggest/Fix the whole sentence, if not what you expect.)